### DISCUSSION TITLE
Not able to delete the latest discount in the product's discounts list

### DISCUSSION THREAD

**Discussion by mohammeddjebloun** [Labs64/NetLicensing-Community#80] [Ideas & Feature Requests]
**Issue description**
Not able to delete the latest discount in the product's discounts list

**How to reproduce**
1. Select a product
2. Click on the Discounts tab
3. Add at least two discounts
4. Click on "Update"
5. Delete all of them
6. Click on "Update"
7. "Success" message appear
7. FIRST ISSUE : none of them is deleted
8. Delete them one by one (delete --> click on update)
9. SECOND ISSUE : They are deleted except the last one, when I click on "update" after deleting it, it appear again.

**Expected behavior**
When I click on update after removing one or multiple discounts, they must be removed.

**Screenshots**
![bug](https://user-images.githubusercontent.com/37653298/125110750-e5565780-e0dc-11eb-8c63-fa817e149b00.gif)

**Comment by v-rudkovskiy** [comment]
PR: https://github.com/Labs64/netlicensing-ui/pull/143

**Comment by r-brown** [comment]
Hello @mohammeddjebloun we fixed this issue and kindly ask you to verify and confirm this from your side.

**Comment by mohammeddjebloun** [comment]
I did multiple tests, the bug seems to be fixed, thanks @r-brown
